olmeiro-orozco/tarea-3/OddOrEven.js

```diff
@@ -0,0 +1,40 @@
+// Task:
+// Given a list of integers, determine whether the sum of its elements is odd or even.
+// Give your answer as a string matching "odd" or "even".
+// If the input array is empty consider it as: [0] (array with a zero).
+
+// Examples:
+// Input: [0]
+// Output: "even"
+
+// Input: [0, 1, 4]
+// Output: "odd"
+
+// Input: [0, -1, -5]
+// Output: "even"
+
+function oddOrEven(array) {
+  let suma = 0;
+  // console.log(array);
+  if (array.length === 1 && array[0] === 0) {
+    return "even";
+  } else if (array.length === 0 && array[0] === 1) {
+    return "odd";
+  } else if (array.length === 0) {
+    return "even";
+  } else {
+    suma = array.reduce((acc, n) => acc + n);
+    // console.log("suma: ", suma);
+    if (suma % 2 === 0) {
+      return "even";
```

Comment:
@OlmeiroOrozco , is not a good practice have so many return sentences , you should use a result variable and only one return at the end

Reply:
I did the change.